kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java

```diff
@@ -164,4 +169,37 @@ public void shouldReturn404ForNonExistingBroker() {
         .expectStatus()
         .isNotFound();
   }
+
+  @Test
+  public void shouldRetrieveTopicConfig() {
+    var topicName = UUID.randomUUID().toString();
+
+    webTestClient.post()
+            .uri("/api/clusters/{clusterName}/topics", LOCAL)
+            .bodyValue(new TopicCreation()
+                    .name(topicName)
+                    .partitions(1)
+                    .replicationFactor(1)
+                    .configs(Map.of())
+            )
+            .exchange()
+            .expectStatus()
+            .isOk();
+
+    List<TopicConfig> configs = webTestClient.get()
+            .uri("/api/clusters/{clusterName}/topics/{topicName}/config", LOCAL, topicName)
+            .exchange()
+            .expectStatus()
+            .isOk()
+            .expectBodyList(TopicConfig.class)
+            .returnResult()
+            .getResponseBody();
+
+    assert configs != null;
+    assert !configs.isEmpty();
+    Assertions.assertNotEquals(null, configs.get(0).getName());
+    Assertions.assertNotEquals(null, configs.get(0).getIsReadOnly());
```

Comment:
I believe there should be a method like `assertNotNull(...)` Could you please use it?

Reply:
This is done